omero/developers/cli/extending.rst

```diff
@@ -1,9 +1,11 @@
 Extensions
 ----------
 
-Plugins can be written and put in the ``lib/python/omero/plugins``
-directory. On execution, all plugins in that directory are registered
-with the |CLI|. Alternatively, the :option:`--path` argument can be used to
+Plugins can be written and put in any directory called ``.../omero/plugins``.
+Add this directory to your ``PYTHON`` path and on execution all plugins in that
+directory are registered with the |CLI| automatically. 
+
+Alternatively, the :option:`--path` argument can be used to
```

Comment:
Newer versions of Sphinx only allow the option markup to be used where the option is included in a definitions list. To save me having to fix this in future, can you just use double back ticks markup here please.